src/fec_base.h

```diff
@@ -755,6 +769,75 @@ void FecCode<T>::decode_lagrange(
         output->set(i, S.get(i));
 }
 
+template <typename T>
+void FecCode<T>::decode_vec_lagrange(
+    vec::Vector<T>* output,
+    const std::vector<Properties>& props,
+    off_t offset,
+    vec::Vector<T>* fragments_ids,
+    vec::Vector<T>* words,
+    vec::Vector<T>* vx,
+    int vx_zero)
```

Comment:
Is `vx_zero` used?

Reply:
yes, it's used in https://github.com/scality/nttec/pull/203/files#diff-827102cc0a2a7d910c0782ad775dd305R144

Reply:
ha, indeed.